tests/test_models/test_utils/test_shape_convert.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmseg.models.utils import nchw2nlc2nchw, nlc2nchw2nlc
+
```

Comment:
We may also add unit test for `nchw_to_nlc` and `nlc_to_nchw`.

Reply:
ok